craco.config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
```

Comment:
The Readme of "Why Did You Render" refers to this issue for adding to create react app
https://github.com/welldone-software/why-did-you-render/issues/154#issuecomment-773905769